src/Apps/Settings/Routes/SavedSearchAlerts/Components/SavedSearchAlertListItem.tsx

```diff
@@ -29,7 +29,7 @@ export const SavedSearchAlertListItem: React.FC<SavedSearchAlertListItemProps> =
   variant,
   onEditAlertClick,
 }) => {
-  const viewAllHref = `${item.href}&search_criteria_id=${item.internalID}`
```

Comment:
Maybe worth noting that we are dropping this because we think the only purpose it served was that jump-scroll interaction, which we [backed out of recently](https://github.com/artsy/force/pull/13410).